src/ngAnimate/animateCss.js

```diff
@@ -749,11 +747,6 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
           options.onDone();
         }
 
-        // Remove the transitionend / animationend listener(s)
-        if (events) {
-          element.off(events.join(' '), onAnimationProgress);
```

Comment:
DISCLAIMER: I haven't investigated if `events` _can_ be empty (or if it will always have atleast one entry), but there's a good chance it can, since someone already noticed it: #13502.